modules/t/test-genome-DBs/homo_sapiens/gene2phenotype/attrib.txt

```diff
@@ -76,7 +76,6 @@
 77	15	altered gene product structure
 78	15	cis-regulatory or promotor mutation
 79	15	increased gene product level
-84      15      decreased gene product level 
```

Comment:
This attrib should not be removed. Maybe it was deleted because there is a space at the end?
`decreased gene product level `